build.sbt

```diff
@@ -1,10 +1,10 @@
 import wartremover.Wart
 import sbt.project
 
-val scala3 = "3.2.0"
+//val scala3 = "3.2.0"
 val scala213 = "2.13.8"
 val scala212 = "2.12.17"
-val supportedScalaVersions = List(scala212, scala213, scala3)
```

Comment:
log4s doesn't work with scala 3

Reply:
Hey, I've just stumbled on this project while trying to restore it in Scala 3 Open Community Build. Can you please give me some context what is actually wrong with log4s? Is it just the issue with non cross-compiling `log4j-api-scala` for Scala 3 yet? You should be able to use safety the Scala 2.13 artifacts here.  I've just tried to use it and it just works. 

```scala
//> using dep "org.log4s::log4s:1.10.0"
//> using dep "org.apache.logging.log4j:log4j-api-scala_2.13:12.0"
//> using dep "org.slf4j:slf4j-simple:2.0.7"

import org.log4s._

@main def Test = 
  val logger = getLogger("foo")
  logger.info("Foo bar")
```

output: 
```scala
> scala-cli run -S 3 test.scala
Compiling project (Scala 3.2.2, JVM)
Compiled project (Scala 3.2.2, JVM)
[main] INFO foo - Foo bar
```